test/suites/integration/httproute_path_match_test.go

```diff
@@ -38,39 +37,33 @@ var _ = Describe("HTTPRoute path matches", func() {
 			service2,
 			deployment2,
 		)
-		time.Sleep(3 * time.Minute) //Need some time to wait for VPCLattice resources to be created
-
-		// Verify VPC Lattice Resource
-		vpcLatticeService := testFramework.GetVpcLatticeService(ctx, pathMatchHttpRoute)
-		Expect(*vpcLatticeService.DnsEntry).To(ContainSubstring(latticestore.LatticeServiceName(pathMatchHttpRoute.Name, pathMatchHttpRoute.Namespace)))
-
-		targetGroupV1 := testFramework.GetTargetGroup(ctx, service1)
-		Expect(*targetGroupV1.VpcIdentifier).To(Equal(os.Getenv("CLUSTER_VPC_ID")))
-		Expect(*targetGroupV1.Protocol).To(Equal("HTTP"))
-		targetsV1 := testFramework.GetTargets(ctx, targetGroupV1, deployment1)
-		Expect(*targetGroupV1.Port).To(BeEquivalentTo(80))
-		for _, target := range targetsV1 {
-			Expect(*target.Port).To(BeEquivalentTo(service1.Spec.Ports[0].TargetPort.IntVal))
-			Expect(*target.Status).To(Or(
-				Equal(vpclattice.TargetStatusInitial),
-				Equal(vpclattice.TargetStatusHealthy),
-			))
-		}
-
-		targetGroupV2 := testFramework.GetTargetGroup(ctx, service2)
-		Expect(*targetGroupV2.VpcIdentifier).To(Equal(os.Getenv("CLUSTER_VPC_ID")))
-		Expect(*targetGroupV2.Protocol).To(Equal("HTTP"))
-		targetsV2 := testFramework.GetTargets(ctx, targetGroupV2, deployment2)
-		Expect(*targetGroupV2.Port).To(BeEquivalentTo(80))
-		for _, target := range targetsV2 {
-			Expect(*target.Port).To(BeEquivalentTo(service2.Spec.Ports[0].TargetPort.IntVal))
-			Expect(*target.Status).To(Or(
-				Equal(vpclattice.TargetStatusInitial),
-				Equal(vpclattice.TargetStatusHealthy),
-			))
-		}
-
-		Eventually(func(g Gomega) {
+		deployments := []*appsv1.Deployment{deployment1, deployment2}
```

Comment:
Just remove the time.Sleep(3 * time.Minute) and move some verification logic in the `Eventually()` block